components/e2etests/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
+enum TestOp {
```

Comment:
Should these go in a common shared crate instead?  #88

Reply:
Ah, I see what you're saying.  Yes, I agree.  I left my comments in the issue.